### PR TITLE
added fauna and kickbyte, and updated some formatting

### PR DESCRIPTION
**Pull request by kdehaan**

### CODE REVIEW

**Comment by ajyong** [comment]
@kdehaan sorry about this, SASS files weren't actually working this earlier today but I think I've fixed that.  Could you fix the conflicts here?  I recommend just deleting the `main.css` and `main.css.map` files, along with everything inside your local `_site` folder before running the `serve` command again.

**Comment by ajyong** [comment]
~Hmm, @kdehaan I'll try my hand at fixing this up~ Nice, we're very close

**Comment by ajyong** [comment]
Those files got moved into a different folder `_sass/` which is why it borked

**Comment by kdehaan** [comment]
@ajyong check out my current version, it should (?) work (?)
the CRLF was probably at least partially my fault, but I've since updated my git globals

**Comment by kdehaan** [comment]
also, not all files have been prettier'd, so there might be some changes on that end as well. This version works fine on my end now, but that doesn't really mean much until you can verify :P

File: _sass/components/_section.scss [hunk 1]
```diff
@@ -39,8 +45,4 @@ header {
     margin-top: (_size(element-margin) * -0.3);
     line-height: 1.5em;
   }
-
-  &.image-header h1 {
```
Comment (ajyong) [inline _sass/components/_section.scss:43]: We'll want to keep this, it keeps the logo's header text hidden.  Rationale for having a hidden header is for SEO purposes

**Review by ajyong — CHANGES_REQUESTED** [review]

**Comment by ajyong** [comment]
> also, not all files have been prettier'd, so there might be some changes on that end as well. This version works fine on my end now, but that doesn't really mean much until you can verify :P

Until we put a pre-deploy hook that explicitly bans un-Prettier'd files, things look fine visually.  Anyways, off to the meeting!

File: _sass/components/_section.scss (hunk 1 above)
Comment (kdehaan) [reply _sass/components/_section.scss:43]: I was not aware that I had edited this at all, was this changed between earlier commit and the revert I did?

Comment (kdehaan) [reply _sass/components/_section.scss:43]: either way will revert to having this again

**Review by kdehaan — COMMENTED** [review]
Made requested changes, sanity check on linux still desired

**Comment by ajyong** [comment]
> Made requested changes, sanity check on linux still desired

I'll check out your branch on my machine and verify

**Comment by ajyong** [comment]
Looks good!  We shouldn't have as much trouble now that everyone's feature branches are based off the Jekyll project.  Thanks for the help!

**Pull request merged**